adapters/spotxchange/usersync_test.go

```diff
@@ -0,0 +1,19 @@
+package spotxchange
+
+import (
+	"testing"
+	"text/template"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewSpotxSyncer(t *testing.T) {
+	temp := template.Must(template.New("sync-template").Parse("//sync.search.spotxchange.com/partner?adv_id=BUYER_ID&uid=BUYER_USER_ID&img=IMG&redir=REDIR_URL"))
```

Comment:
how does this prebid server fill out the tokens? the redir param on here is also a bit weird unless if we are trying to dual sync, perhaps we should talk in person?

Reply:
Probably. I could use some help with a few other pieces to make sure I'm getting things right.